indico/htdocs/js/indico/jquery/trackrolewidget.js

```diff
@@ -0,0 +1,94 @@
+/* This file is part of Indico.
```

Comment:
`track_role_widget.js`? Those filenames without any separators are awful, especially now that many editors have smart search where `trwi` would find `track_role_widget` but not `trackrolewidget`...

I know it's consistent with the ones we already have but I'd also consider renaming these at some point...